Automattic/pocket-casts-ios
# Transcripts: Fallback to other transcript format if main one fails

| 📘 Part of: #1848  |  
|:---:|

This PR changes the way transcripts are loaded so that if the preferred format of transcripts is not working correctly, empty or parsing error, we will try to load a different transcript format.

One of the examples of this happening is on the Chaos Radio transcripts where while they an available `vtt` transcript it's empty so we can go for the next on the list that is `podcast index json` format and it will load it.

## To test

- Start the app
- Ensure that you have the `transcripts` and `newShowNotesEndpoint` FF enabled
- Open any episode from [Chaos Radio](https://pca.st/chaosradio)
- Tap Play
- Open the full screen player
- Tap on transcripts
- Check that the transcript information is show correctly

## Checklist
